src/modules/form.js

```diff
@@ -0,0 +1,23 @@
+export default function form() {
```

Comment:
- [ ] Please use arrow functions. This is a project requirement. Arrow functions are also more compact and fit it well with the functional programming paradigm. Update all functions in form.js, score.js & title.js files. 👍🏾

Reply:
tky for your comments and your patience, I change all functions for arrow functions